kafka-rest/src/main/java/io/confluent/kafkarest/controllers/SchemaRecordSerializerImpl.java

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
Has there been changes in this that I need to review. With the move from `SchemaRecordSerializer` to `SchemaRecordSerializerImpl`, everything appears as new, so it's hard to see changes.

Reply:
![image](https://user-images.githubusercontent.com/87994408/148418481-80e45eb2-336a-4ddc-aa9f-48c3910d8f90.png)